alacritty.yml

```diff
@@ -497,45 +501,79 @@
 #   - ScrollToTop
 #   - ScrollToBottom
 #   - ClearHistory
+#       Remove the terminal's scrollback history.
 #   - Hide
+#       Hide the Alacritty window.
 #   - Minimize
+#       Minimize the Alacritty window.
 #   - Quit
+#       Quit Alacritty.
 #   - ToggleFullscreen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty.
 #   - ClearLogNotice
+#       Clear Alacritty's UI warning and error notice.
 #   - ClearSelection
+#       Remove the active selection.
 #   - ReceiveChar
 #   - None
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Opens URLs at the cursor location with the launcher configured in `url.launcher`.
 #   - Up
+#       Move the vi mode cursor up by one line.
 #   - Down
+#       Move the vi mode cursor down by one linee.
 #   - Left
+#       Move the vi mode cursor one character left.
 #   - Right
+#       Move the vi mode cursor one character right.
 #   - First
+#       Move to the start of line, including wrapped lines.
 #   - Last
+#       Move to the end of line, including wrapped lines.
 #   - FirstOccupied
+#       Move to the first non-empty cell.
 #   - High
+#       Move to the top of the screen.
 #   - Middle
+#       Move to the middle of the screen.
 #   - Low
+#       Move to the bottom of the screen.
 #   - SemanticLeft
+#       Move to start of the previous semantically separated word.
 #   - SemanticRight
+#       Move to start of the next semantically separated word.
 #   - SemanticLeftEnd
+#       Move to end of the previous semantically separated word.
 #   - SemanticRightEnd
-#   - WordRight
+#       Move to end of the next semantically separated word.
 #   - WordLeft
-#   - WordRightEnd
+#       Move to start of the previous whitespace separated word.
+#   - WordRight
+#       Move to start of the next whitespace separated word.
 #   - WordLeftEnd
+#       Move to end of the previous whitespace separated word.
+#   - WordRightEnd
+#       Move to end of the next whitespace separated word.
 #   - Bracket
+#       Move the vi mode cursor to the next character that matches the bracket at the cursor's
+#       current location.
 #   - ToggleNormalSelection
 #   - ToggleLineSelection
 #   - ToggleBlockSelection
 #   - ToggleSemanticSelection
+#       Toggle selecting the semantic string under the cursor.
+#       See semantic_escape_chars to change boundary detection.
 #   - SearchNext
+#       Once a search has completed, jump to the beginning of the next match.
```

Comment:
I don't like the wording of this, since the search is still ongoing. Just the search regex input has been completed.
Though I'm not entirely sure how to word it any better.

Reply:
I'll try "Once a search term has been entered,"